### PR TITLE
chore: update privacy policy and TOU

### PR DESCRIPTION
**Pull request by mantariksh**

### CODE REVIEW

File: pages/privacy.md [hunk 1]
```diff
@@ -4,21 +4,113 @@ permalink: /privacy/
...
+		
+    9.2. need more information on or access to data which you have provided to us in the past.
+
+    **Definitions**
+10. In this Privacy Statement, “Public Sector Entities” means the Government (including its ministries, departments and organs of state) and public authorities (such as statutory boards) and “personal data” shall have the same meaning as its definition in the Personal Data Protection Act 2012 (No. 26 of 2012).
+
+This version of the Privacy Statement is dated 28/11/2022.
+
+### Annex
+1. **Name of Service**: sgID mobile
```
Comment (kwajiehao) [inline pages/privacy.md:75]: How did we arrive at the name of the service? Is this something that legal decided?

Also, in the schedule section of the `Terms of Use`, the service name is given as `sgID (mobile)` with parentheses. Should this be consistent with the name here?

File: pages/privacy.md [hunk 2]
```diff
@@ -4,21 +4,113 @@ permalink: /privacy/
...
+2. **Types of data requested/collected**
```
Comment (kwajiehao) [inline pages/privacy.md:76]: We are also getting `HDB Type` and `Housing Type` as of a couple weeks ago

**Review by kwajiehao — CHANGES_REQUESTED** [review]
left some comments

File: pages/privacy.md (hunk 1 above)
Comment (mantariksh) [reply pages/privacy.md:75]: just copy-pasting from what Veena sent, which is what GT legal approved. I don't know what the considerations are or if the name makes a difference. would you like me to check?

Comment (kwajiehao) [reply pages/privacy.md:75]: I'll start a thread in the sgid channel, but i think we can just go ahead with this first

File: pages/privacy.md (hunk 2 above)
Comment (mantariksh) [reply pages/privacy.md:76]: updated!

**Review by kwajiehao — APPROVED** [review]
lgtm

**Pull request merged**